st2client/st2client/shell.py

```diff
@@ -78,6 +78,11 @@
     'debug': ['cli', 'debug']
 }
 
+# A list of command classes for which automatic authentication should be skipped.
+SKIP_AUTH_CLASSES = [
+    'TokenCreateCommand'
```

Comment:
@manasdk So let me know what you think, i'm fine with changing it back to checking for "auth" argv.